vote/cnab/bundle.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I believe the convention is that this `bundle.json` file is placed at root of bundle (so, `vote/bundle.json` in this particular case.)

Reply:
I'm a bit confused about this too. All the other bundles here follow the convention you mention, but the examples at duffle (e.g. https://github.com/deislabs/duffle/tree/master/examples/helloazure) follow the convention of having it inside the CNAB folder.  With this configuration duffle was able to build my bundle.

What's the right convention to follow?

Reply:
we probably need to update the examples in duffle. The bundles repository is more up-to-date.